ClientCore/I18N/Locale.cs

```diff
@@ -0,0 +1,269 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using System.Reflection.Metadata;
+using Rampastring.Tools;
+using Rampastring.XNAUI.XNAControls;
+
+namespace ClientCore.I18N;
```

Comment:
I18N is different from L10N

Reply:
Intentional.

Reply:
Localization: Providing multiple translations.
Internalization: Always use UTC time, english text for logs, etc.

Totally different

Reply:
Wikipedia:
> Internationalization is the process of designing a software application so that it can be adapted to various languages and regions without engineering changes. Localization is the process of adapting internationalized software for a specific region or language by translating text and adding locale-specific components.

> Localization (which is potentially performed multiple times, for different locales) uses the infrastructure or flexibility provided by internationalization (which is ideally performed only once before localization, or as an integral part of ongoing development).[[3]](https://en.wikipedia.org/wiki/Internationalization_and_localization#cite_note-3)

Reply:
You are right. I18N is fine.